changelog/6512.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Add support for Python 3.11.
+
+The deprecated `cgi.parse_header` is now available as
+`sunpy.util.net.parse_header`.
```

Comment:
I'm not sure we want to be advertising this publicly in a changelog since the new vendored code isn't public API? I'd suggest just removing the changelog altogether.

Reply:
We document it in the Fido dev guide so it's public api to my mind.